code/workspaces/web-app/src/containers/adminListUsers/DataStores.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
-import { TYPE_NAME_PLURAL } from '../dataStorage/DataStorageContainer';
+import { TYPE_NAME, TYPE_NAME_PLURAL } from '../dataStorage/DataStorageContainer';
 import ResourceStackCards from './ResourceStackCards';
-import { STORAGE_TYPE_NAME } from '../dataStorage/storageTypeName';
```

Comment:
Would it make sense to have the singular and plural type names coming from the same place?